UI/src/view/SpeedLimitLayer.js

```diff
@@ -291,6 +291,10 @@ window.SpeedLimitLayer = function(params) {
     eventListener.listenTo(eventbus, 'speedLimits:drawSpeedLimitsHistory', drawSpeedLimitsHistory);
     eventListener.listenTo(eventbus, 'speedLimits:hideSpeedLimitsHistory', hideSpeedLimitsHistory);
     eventListener.listenTo(eventbus, 'speedLimits:showSpeedLimitsHistory', showSpeedLimitsHistory);
+    eventListener.listenTo(eventbus, 'toggleWithRoadAddress', function(){
+      if(applicationModel.getSelectedLayer() == layerName)
+        me.refreshView();
+    });
```

Comment:
In this case, it's more pretty if we create a function bellow and not at eventListener, only for use the same idea the others "listenTo's".